_includes/navbar.html

```diff
@@ -18,7 +18,7 @@
 
         <div class="collapse navbar-collapse" id="odata-navbar-collapse">
 
-            <ul class="nav navbar-nav" role="navigation">
+            <ul class="nav navbar-nav" role="list">
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role maybe have the role navigation.

Reply:
maybe I can add it on the div element